queries/run_query.py

```diff
@@ -14,23 +16,42 @@ def main():
     """
     Script to export validated JSON-LD data out of a RDF store.
     """
+
+    # Define location to which results json will be saved from arg. Defaults to ./results/ if no
+    # argument provided.
+    def parse_args():
+        parser = argparse.ArgumentParser()
+        parser.add_argument(
+            "--loc",
```

Comment:
`--output`, `--output-dir`, or `-o` tends to be the default for command line programs for output locations, but this is an improvement on the previous functionality and not a deal breaker for me.